src/contracts/core/DelegationManager.sol

```diff
@@ -181,12 +181,19 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
      * @dev Note that it is assumed that a staker places some trust in an operator, in paricular for the operator to not get slashed; a malicious operator can use this function
      * to inconvenience a staker who is delegated to them, but the expectation is that the inconvenience is minor compared to the operator getting purposefully slashed.
      */
-    function forceUndelegation(address staker) external returns (bytes32) {
+    function forceUndelegation(address staker) external returns (bytes32, bytes32) {
         address operator = delegatedTo[staker];
         require(staker != operator, "DelegationManager.forceUndelegation: operators cannot be force-undelegated");
         require(msg.sender == operator || msg.sender == _operatorDetails[operator].delegationApprover,
             "DelegationManager.forceUndelegation: caller must be operator or their delegationApprover");
-        return strategyManager.forceTotalWithdrawal(staker);
+        
+        bytes32[] memory queuedWithdrawals = new bytes32[](2);
```

Comment:
seems like you meant for this to be the return values but aren't using it anymore? would recommend deleting